sorts/stalin_sort.py

```diff
@@ -0,0 +1,37 @@
+"""
+This is a pure Python implementation of the stalin sort algorithm,
+inspired by the dictator of the former soviet union Josef Stalin.
+
+This sorting algorithm sorts a collection of elements by deleting
+every element that is not in order resulting in a sorted collection.
+
+For manual testing run:
+python stalin_sort.py
+"""
+
+
+def stalin_sort(collection):
```

Comment:
Please add type hints

Reply:
Thank you for your kind and helpful advices! I'll try to implement all of them, but I have a question for this one:
Should I add type hints to the whole script or is the functions head enough?